src/Kirel.Identity.Core/Validators/KirelUserCreateDtoValidator.cs

```diff
@@ -40,7 +40,7 @@ public KirelUserCreateDtoValidator(UserManager<TUser> userManager, RoleManager<T
             .Matches(@"^(?=.*[a-zA-Z]{1,})(?=.*[\d]{0,})[a-zA-Z0-9.]{4,20}$").WithMessage("Username can only contains letters, numbers and dots")
             .Must((dto, _) => UserNameUnique(dto.UserName, out message)).WithMessage(_ => message);
         RuleFor(dto => dto.Email)
-            .Matches(@"^[\w._-]{4,}@+\w{2,}(.com|.co|.uk|.ru)$").WithMessage("Email must be abc@def.com|ru")
+            .EmailAddress().WithMessage("Email must be abc@def.com|ru")
```

Comment:
Is it possible that there is a standard message because there is a standard rule?

Reply:
There is localized message. I changed our message to the same in english for now